src/doc/unstable-book/src/compiler-flags/sanitizer.md

```diff
@@ -865,6 +867,69 @@ WARNING: ThreadSanitizer: data race (pid=10574)
   Location is global 'example::A::h43ac149ddf992709' of size 8 at 0x5632dfe3d030 (example+0x000000bd9030)
 ```
 
+# RealtimeSanitizer
+RealtimeSanitizer detects blocking calls in realtime contexts. If enabled,
+it won't sanitize the whole program, it needs to be enabled explicitly.
```

Comment:
Suggestion for this second sentence:

"Usage of RTSan is on a function-by-function basis. The sanitizer only inspects  code between calls of `__rtsan_realtime_enter()` and `__rtsan_realtime_exit()`. Any function that is subject to real-time constraints must put `__rtsan_realtime_enter()` at its entry point, and `__rtsan_realtime_exit()` at each exit point.